roles/openshift_facts/vars/Fedora.yml

```diff
@@ -0,0 +1,7 @@
+---
+required_packages:
+  - iproute
+  - python3-dbus
+  - python-six
+  - PyYAML
+  - yum-utils
```

Comment:
I faced yet another problem with this package: when `yum-utils` is already installed all the process failed with:
```
TASK [openshift_facts : Ensure various deps are installed] *********************
failed: [ec2-54-83-174-64.compute-1.amazonaws.com] (item=[u'iproute', u'python3-dbus', u'python-six', u'PyYAML', u'yum-utils']) => {
    "failed": true, 
    "item": [
        "iproute", 
        "python3-dbus", 
        "python-six", 
        "PyYAML", 
        "yum-utils"
    ], 
    "module_stderr": "Package iproute-4.4.0-3.fc24.x86_64 is already installed, skipping.\nPackage python3-dbus-1.2.4-1.fc24.x86_64 is already installed, skipping.\nPackage python-six-1.10.0-2.fc24.noarch is already installed, skipping.\nPackage PyYAML-3.11-12.fc24.x86_64 is already installed, skipping.\nTraceback (most recent call last):\n  File \"/tmp/ansible_br2rb44o/ansible_module_dnf.py\", line 375, in <module>\n    main()\n  File \"/tmp/ansible_br2rb44o/ansible_module_dnf.py\", line 369, in main\n    ensure(module, base, params['state'], params['name'])\n  File \"/tmp/ansible_br2rb44o/ansible_module_dnf.py\", line 324, in ensure\n    base.do_transaction()\n  File \"/usr/lib/python3.5/site-packages/dnf/base.py\", line 591, in do_transaction\n    self._trans_error_summary(errstring))\ndnf.exceptions.Error: Transaction check error:\n  package yum-utils-1.1.31-509.fc24.noarch is already installed\n\nError Summary\n-------------\n\n", 
    "module_stdout": ""
}

MSG:

MODULE FAILURE
```
To overcome this I had to remove `yum-utils` from this list.